packages/erc3k/contracts/ERC3000Data.sol

```diff
@@ -31,9 +33,11 @@ library ERC3000Data {
 
     struct Config {
         uint256 executionDelay;
+        // Note: I'd call these variables "xCollateral" or rename the struct to "Deposit", I prefer the first option
```

Comment:
I was doubting this. I personally like deposit a bit more. Thoughts @dizzypaty @mariapao @lkngtn?

In any case I agree on consistency